### PR TITLE
Separate Function for readability

### PR DESCRIPTION
**Pull request by SaraVieira**
**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [x] refactoring / style
- [ ] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [ ] Yes, I've updated the documentation
- [x] Not relevant

**Summary**
This is my opinion on the separation of functions from the main file because the main file was getting too big and confusing

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import accepts from 'attr-accept'
+
+export const supportMultiple =
```
Comment (okonet) [inline src/utils/index.js]: I really prefer `function` declarations for functions unless they are in the clojure. Can you live this that and make me happy again™? ;)

File: src/utils/styles.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    width: 200,
+    height: 200,
+    borderWidth: 2,
+    borderColor: '#666',
+    borderStyle: 'dashed',
+    borderRadius: 5
+  }
+}
+
+export default styles
```
Comment (okonet) [inline src/utils/styles.js]: Why doing it on a separate line?

File: src/index.js [hunk 3]
```diff
@@ -2,26 +2,17 @@
...
-  typeof document !== 'undefined' && document && document.createElement
-    ? 'multiple' in document.createElement('input')
-    : true
-
-function fileAccepted(file, accept) {
-  // Firefox versions prior to 53 return a bogus MIME type for every file drag, so dragovers with
-  // that MIME type will always be accepted
-  return file.type === 'application/x-moz-file' || accepts(file, accept)
-}
+import getDataTransferItems from './utils/getDataTransferItems'
```
Comment (okonet) [inline src/index.js]: Should this also go to `utils.js`?

File: src/utils/index.js (hunk 1 above)
Comment (SaraVieira) [inline src/utils/index.js]: yeah , sure but that one is not a function , just and if statement :p

File: src/utils/styles.js (hunk 2 above)
Comment (SaraVieira) [inline src/utils/styles.js]: Habit ! Fixed 👍

File: src/index.js (hunk 3 above)
Comment (SaraVieira) [inline src/index.js]: Yeah , I'll pass it

**Comment by SaraVieira** [comment]
@okonet All done

**Pull request merged**